.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    # Run daily at 430 am PT
+    - cron: '30 11 * * *'
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why do you need this here?

Reply:
the action interacts with & controls repo issues - labelling, commenting, closing. The official actions/stale doc examples don't actually work (???) but commenters in the repo say you need `github_token` and/or `permissions:` key to work

Reply:
Update, tested and confirmed that
```
permissions:
  issues: write
```

is needed but the token is not, so I've remove the token part